### PR TITLE
feat(jdbc,runner-memory): send a FAILED worker task result when we ca…

### PR DESCRIPTION
**Pull request by loicmathieu**
…nnot create it from the executable task

### CODE REVIEW

File: jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java [hunk 1]
```diff
@@ -478,8 +480,15 @@ private void sendWorkerTaskResultForWorkerTaskExecution(Execution execution, Wor
...
             maybeWorkerTaskResult.ifPresent(workerTaskResult -> this.workerTaskResultQueue.emit(workerTaskResult));
         } catch (Exception e) {
-            // TODO maybe create a FAILED Worker Task Result instead
             log.error("Unable to create the Worker Task Result", e);
+            // we send a fail worker task result to end the flow
+            this.workerTaskResultQueue.emit(
+                WorkerTaskResult.builder()
+                    .taskRun(taskRun.withState(State.Type.FAILED).withAttempts(
+                        Collections.singletonList(TaskRunAttempt.builder().state(new State().withState(State.Type.FAILED)).build())
+                    ))
```
Comment (brian-mulier-p) [inline jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java:489]: Please merge to previous attempts instead of erasing previous states history

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]
Just merge failed attempts with previous ones and LGTM (change test to reflect this change)

File: jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java (hunk 1 above)
Comment (loicmathieu) [reply jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java:489]: Attempts are only handled by the Worker when retrying a task.

Here it's a "fake" WorkerTaskResult that we fail to create from the executable task, there cannot be any existing attempt (and retrying to create the WorkerTaskResult has no meaning).

**Review by brian-mulier-p — APPROVED** [review]
LGTM

**Pull request merged**